tools/ci/build.rs

```diff
@@ -0,0 +1,4 @@
+#[allow(clippy::disallowed_macros)]
+fn main() {
+    println!("cargo:rustc-env=TARGET={}", std::env::var("TARGET").unwrap_or_default());
```

Comment:
what's this part for?

Reply:
(honestly I'm not 100% sure that we even need the target param.. I think the default on each platform might be the one we want anyway.. but we should keep it for now just for parity with the original CI).